_posts/2020-11-19-Subnero-at-SIWW-online.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: Subnero attends SIWW online
+banner : images/banner-pulse-standard.jpg
+date:  2020-11-19
+categories: wnc
+thumbnail: images/pulse-thumbnail-siww-online.jpg
+---
+
+Due to the pandemic, Singapore International Water Week has been postponed to July 2021.
+Instead of waiting until then, the organisors arranged a virtual show.
+
+Subnero had a virtual booth with personnel available on line, permanently, to attend any visitors' queries.
```

Comment:
Can we mention what was displayed in the booth and have a link to the SWAN White paper and the modem brochure?

Reply:
Good idea, I also need to add the link to the website to SIWW.